### PR TITLE
adjust pgbouncer settings

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph

### CODE REVIEW

**Comment by snopoke** [comment]
I think the total connections (default_pool_size + reserve_pool_size) should always be below the max_connections setting for postgresql.

File: ansible/vars/dev.yml [hunk 1]
```diff
@@ -47,6 +47,12 @@ postgres_users:
     username: "{{ localsettings.PG_DATABASE_USER }}"
     password: "{{ localsettings.PG_DATABASE_PASSWORD }}"
 
+postgres_config:
```
Comment (TylerSheffels) [inline ansible/vars/dev.yml]: Isn't this a huge decrease in max_connections? Did you use a different number on prod?

Comment (benrudolph) [inline ansible/vars/dev.yml]: this is just for dev.yml. presumably the real ones will be higher

**Comment by benrudolph** [comment]
dont' think anything is blocking here, so i'll go ahead and merge since @TylerSheffels is independent of this PR

**Pull request merged**